marketplacetests/tests/manifest.ini

```diff
@@ -25,13 +25,12 @@ offline = false
 
 [test_marketplace_login_from_app_details_page.py]
 
-[test_marketplace_login_from_my_apps.py]
-
 [test_marketplace_make_an_in_app_payment.py]
 
 [test_marketplace_purchase_app.py]
 
 [test_marketplace_search_and_install_app.py]
+disabled = https://bugzilla.mozilla.org/show_bug.cgi?id=1157317
```

Comment:
To save having to visit this URL could you change the value to "Bug 1157317 - Fake apps should have unique names" or a similar summary if the bug summary is not sufficient.